site/data/sidebar.yml

```diff
@@ -105,8 +105,7 @@
       draft: true
     - title: Back to top
       draft: true
-    - title: Badge
-      draft: true
+    - title: Badges
```

Comment:
Maybe we should stick to the Bootstrap one or change all the urls linking to this page